Sources/Commands/SwiftTestTool.swift

```diff
@@ -210,6 +210,9 @@ public class SwiftTestTool: SwiftTool<TestToolOptions> {
             let toolchain = try getToolchain()
             let testPath = try buildTestsIfNeeded(options, graph: graph)
             var ranSuccessfully = true
+            let buildParameters = try self.buildParameters()
+
+            try fileManager.removeFileTree(buildParameters.buildPath.appending(component: "codecov"))
```

Comment:
Some points here:

* We should also add comments here explaining _why_ are we deleting the codecov directory. This will help the future maintainer in understanding why we're doing this.
* I think we should add a new property in BuildParameters to return the path of the "codecov" directory. That way we can avoid using raw strings every time we need to get the codecov directory.
* We should only delete this directory if codecov option is enabled.

Reply:
done